funding/config_rpctest.go

```diff
@@ -7,5 +7,5 @@ import "time"
 func init() {
 	// For itest, we will use a much shorter checking interval here as
 	// local communications are very fast.
-	checkPeerFundingLockInterval = 10 * time.Millisecond
+	checkPeerFundingLockInterval = 10 * time.Millisecond //nolint:gomnd
```

Comment:
We've silenced the magic number detector, so this can be removed. The other changes in this commit are nice to have though :tada:

Reply:
done!